src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.application.name=user-data-store
 
-spring.datasource.url=${USER_DATA_STORE_DATASOURCE_URL:jdbc:postgresql://host.docker.internal:5432/powerauth}
+spring.datasource.url=${USER_DATA_STORE_DATASOURCE_URL:jdbc:postgresql://localhost:5432/powerauth}
```

Comment:
Is this change needed?

Reply:
@banterCZ Not in theory, just trying to set some "standard". In Dockerized projects like pac and pa-tests we do have 
`spring.datasource.url=jdbc:postgresql://localhost:5432/powerauth` here.

Reply:
If so, it does not belong to this PR anyway.

Reply:
Arlight